pom.xml

```diff
@@ -1023,6 +1051,11 @@
                                             <goal>analyze-duplicate</goal>
                                             <goal>analyze-only</goal>
                                         </goals>
+                                        <configuration>
```

Comment:
This part is just for Eclipse. It shouldn't have configuration.

(I really dislike how it's in the POM and wish it could go elsewhere)